### DISCUSSION TITLE
Add option for entiering durations with the clock face instead of slider

### DISCUSSION THREAD

**Discussion by bsaunder2002** [jithware/brethap#87] [Ideas]
I like to set the duration on the 4-7-8 to 5 minutes.  Since the slider covers 1 min to a very long time and the slider size adjusts via screen size, on a small screen, it's hard to pick 5mins not 3 or 4 or 6 or 7.  They're all a pixel or two away.
The clockface where you drag around the hour hand and minute hand to specify a time...probably drop the hour hand is much easier to use on a small screen.
I'd envision a button next to the slider that brings up the clock face option.

**Comment by jithware** [comment]
Don't see any "built-in" [flutter widgets](https://pub.dev/packages?q=clock) to accomplish this. Would take a significant effort to build one from scratch. As a work around for smaller screens, you can rotate your screen sideways (landscape mode) to have a longer slider.

**Comment by bsaunder2002** [comment]
I don't write code (often) nor have any clue about the flutter api but just googling to see what was available in flutter, came across this.  It was a good example of what I like.  It is personal preference and I'm not trying to push mine...just an example.
https://gitanswer.net/flutter-flutter-timepicker-is-not-consistent-with-native-android-time-picker-dart-700630275

**Reply by jithware** [reply]
Whatever potential time picker used, it would also need a way to select seconds as well. For example, the [physiological sigh](https://github.com/jithware/brethap/blob/987a0fecbe5b5f918d008c5d802e22e821b2b4da/lib/constants.dart#L99) preset is under a minute and only uses seconds. Perhaps it would be a good opportunity for you to learn flutter by creating a time picker widget and publishing it to pub.dev ;)
